### ISSUE TITLE
persistent volume claim doesn't recognize storage_class_name

### ISSUE THREAD

**Issue by matthiaskubik** [terraform-providers/terraform-provider-kubernetes#95]
Hi there,

### Terraform Version
Terraform v0.9.8

### Affected Resource(s)
- kubernetes_persistent_volume_claim
  - storage_class_volume

### Terraform Configuration Files
```hcl
provider "kubernetes" {
   host = "https://localhost"
}

resource "kubernetes_persistent_volume_claim" "example" {
  metadata {
    name = "exampleclaimname"
  }
  spec {
    access_modes = ["ReadWriteMany"]
    resources {
      requests {
        storage = "5Gi"
      }
    }
    volume_name = "p-vol"
    storage_class_name = "aa"
  }
}
```

### Debug Output
terraform plan
1 error(s) occurred:

* kubernetes_persistent_volume_claim.example: spec.0: invalid or unknown key: storage_class_name

### Expected Behavior
storage_class_name should be accepted as documented

### Actual Behavior
* kubernetes_persistent_volume_claim.example: spec.0: invalid or unknown key: storage_class_name

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform plan`

**Comment by radeksimko** [comment]
This was addressed in #111 which is part of the latest release (`1.1.0`). Thanks for the report.

**Comment by SwitchedToGitlab** [comment]
This issue is not addressed, and persists in Terraform 0.11.14 with Kubernetes provider version 1.6.2
